config/routes.ignore.yml

```diff
@@ -0,0 +1,2 @@
+DashboardController:
+  - login
```

Comment:
If this ignore file is only for specs, then I suggest it lives in the spec directory instead...maybe `spec/config` or `spec/data`?

Reply:
:+1: